docs/en/sql_reference/aggregate_functions/combinators.md

```diff
@@ -51,33 +51,107 @@ Converts an aggregate function for tables into an aggregate function for arrays
 
 ## -OrDefault {#agg-functions-combinator-ordefault}
 
-Fills the default value of the aggregate function’s return type if there is nothing to aggregate.
+This combinator replaces returned value with default one, if the aggregate function does not have values to calculate. Works with different aggregate functions.
+`-OrDefault` can be combined with other combinators.
+
+**Syntax** 
+
+``` sql
+<aggFunction>OrDefault(x)
+```
+
+**Parameters**
+
+- `x` - Parameter of the aggregate function. A type of the parameter depends on the specific aggregate function.
+
+**Returned values** 
+ 
+Returns default value of the aggregate function’s return type if there is nothing to aggregate.
+
+Type depends on the specific aggregate function.
+
+**Example**
+
+Query:
 
 ``` sql
 SELECT avg(number), avgOrDefault(number) FROM numbers(0)
 ```
 
+Result:
+
 ``` text
 ┌─avg(number)─┬─avgOrDefault(number)─┐
 │         nan │                    0 │
 └─────────────┴──────────────────────┘
 ```
 
+Also `-OrDefault` can be used with a few combinators. It is useful when the aggregate function does not accept the empty input.
+
+Query:
+
+``` sql
+SELECT avgOrDefaultIf(x, x > 10)
+FROM
+(
+    SELECT toDecimal32(1.23, 2) AS x
+)
+```
+
+Result:
+
+``` text
+┌─avgOrDefaultIf(x, greater(x, 10))─┐
+│                              0.00 │
+└───────────────────────────────────┘
+```
+
+**See also**
+
+- [Functions For Working With External Dictionaries](../functions/ext_dict_functions.md#ext_dict_functions-other) - `-OrDefault` is used with functions for working with external dictionaries.
+
 ## -OrNull {#agg-functions-combinator-ornull}
 
-Fills `null` if there is nothing to aggregate. The return column will be nullable.
+This combinator replaces returned value with nullable one, if the aggregate function does not have values to calculate. Works with different aggregate functions. 
```

Comment:
Nullable is a data type, but we talk about the specific value NULL. The function can only return NULL in case of empty input data set and/or it can convert the return data type in the Nullable data type.

Check it please and update the desctiption.